plugins/processors/awsentity/processor.go

```diff
@@ -110,6 +111,10 @@ func newAwsEntityProcessor(config *Config, logger *zap.Logger) *awsEntityProcess
 	}
 }
 
+func (p *awsEntityProcessor) processLogs(_ context.Context, ld plog.Logs) (plog.Logs, error) {
```

Comment:
This function is a no-op for now since this PR is focused on adding `awsentity` processor to the pipeline. Will be expanded in later PRs to add entity attributes for log pipelines